soc/realtek/ec/common/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2024 Realtek Semiconductor Corporation, SIBG-SD7
+#
+
+zephyr_include_directories(.)
+
+math(EXPR adjustment "${CONFIG_REALTEK_RTS5912_BOOTROM_HEADER_LOAD_ADDRESS}" OUTPUT_FORMAT DECIMAL)
```

Comment:
`if(CONFIG_REALTEK_RTS5912_BOOTROM_HEADER)` ?

Reply:
Add this into commit